### DISCUSSION TITLE
The Delete Key

### DISCUSSION THREAD

**Discussion by coreyog** [community/community#10835] [Copilot Conversations]
Deleting code using the delete key instead of backspace can be confusing while Github Copilot gives you recommendations after each key press. The particular section of code I was working with resulted in a multiline recommendation in the middle of an existing line of code. Since a lot of whitespace was virtually injected into the editor, it was difficult to tell which character would actually be deleted. If I pressed escape, and then delete, the recommendation would return. Pressing the delete key should stop recommending a code snippet in the same way pressing escape does. Or maybe be configurable?

**Comment by jansorg** [comment]
Thanks! Could you add which editor you're using, please?

**Reply by coreyog** [reply]
VSCode 1.63.2 on Big Sur
